Applications/Classification/README.md

```diff
@@ -1,3 +1,89 @@
-# Classifciation
+# Classification
 
+This example demonstrate the transfer learning with mobilenetv2 which pre-trained for imagenet. "mobilenetv2.tflite" is used for feature extractor which is removed last fully connected layer manually.
 
+After feature extraction, it is used for training fully connencted layer. To take the feature as an input, we add input layer before fully connected layer.
+
+Total number of data for training is 10 (number of class) * 100 (number of image for each class)
+
+This example has two options to train.
+### 1. Train with training set files ( main.cpp )
+   - This option is activated if [ DataSet ] section is configured in configuration file.
+   - This assumes that there are training set which saved as an file.
+     For ith image, feature data (62720 x sizeof(float)) + label (10 x sizeof(float)) at i x ((62720 x sizeof(float) + label(10 x sizeof(float)) position.
+   - The input image (bmp format : 32x32x3) is stored as in ExtractFeatures() at "data_path" directory defined as argv #1.
+   ```
+     string total_label[10] = {"airplane", "automobile", "bird", "cat", "deer",
+                            "dog", "frog", "horse", "ship", "truck"};
+```
+   - Training dataset files are generated 'getMiniBatch()' function in main.cpp.
+   - Once it generates the dataset files, it start to read and feed the buffer to train.
+
+### 2. Train with training data generator ( main_func.cpp )
+   - This option is activated if there is no [ DataSet ] section is defined.
+   - Expect to get the generator function whcih defined by user.
+   - This option just use training data set file which is generated by option 1.
+   - training data set generator is getMiniBatch_train(),  getMiniBatch_val() for validation. test data set is not used.
+
+Classification.ini file (res directory) is input which has configuration and hyper-parameters of network for nntrainer.
+
+## How to run examples
+
+### Preparing NNTrainer
+* https://github.com/nnstreamer/nntrainer/blob/master/docs/getting-started.md
+
+### Write Configuration file
+
+you can find detail explanation about each keyward in
```

Comment:
keyward -> keyword
I recommend to use some dictionary based to validate this document once for these minor errors.